### DISCUSSION TITLE
可以做到只刷新hosts文件的一部分内容么？

### DISCUSSION THREAD

**Discussion by liuzhcn** [oldj/SwitchHosts#803] [Q&A]
比如github的dns解析是定期更新的，现在也提供了远程的hosts内容。
但是因为一些原因，hosts要一直保持对其他一些域名的解析。

如果做到在保持其他域名解析设定不变的情况下，指定一部分刷新远程的内容呢？

**Comment by oldj** [comment]
可以新建多个方案，同时开启？需要刷新的是远程方案，其他规则可以放普通本地方案里。

**Reply by liuzhcn** [reply]
可以支持多方案合并到hosts里面？我还没注意这个功能。

**Reply by DShuoTypoLE** [reply]
Can you introduce the detailed steps

**Reply by DShuoTypoLE** [reply]
> Can you introduce the detailed steps
oh ha,I've solved the problem that you can do something similar to this!
Of course we need to change the write mode to overwrite
![image](https://github.com/user-attachments/assets/833fc001-ad05-4acc-bee6-852eee28fc93)
